production/install

```diff
@@ -1855,7 +1921,12 @@ echo "[*] Adding MySQL configuration"
 case $OS in
 
     FreeBSD)
-        osSudo "${ROOT_USER}" service mysql-server onestart
+        if ! osSudo "${ROOT_USER}" service mysql-server status >/dev/null 2>&1; then
+            echo "MySQL is not running. Starting..."
+            osSudo "${ROOT_USER}" service mysql-server onestart
+        else
+            echo "MySQL is already running."
+        fi
```

Comment:
It might make more sense to change this to a loop (including replacing the `sleep 10` below), where you run the onestart command with `|| true` at the end of the commend to not exit this script if it fails for some reason (like if mysql is already running), and then `sleep 1` inside the loop checking the status and continuing after you verify that mysql has indeed started